src/providers/Web3ModalProvider.tsx

```diff
@@ -0,0 +1,57 @@
+import '@rainbow-me/rainbowkit/styles.css';
+
+import React from 'react';
+
+import { RainbowKitProvider, connectorsForWallets, darkTheme } from '@rainbow-me/rainbowkit';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { rainbowWallet, walletConnectWallet, injectedWallet } from '@rainbow-me/rainbowkit/wallets';
+import { WagmiProvider, http, createConfig } from 'wagmi';
+import { localhost } from 'wagmi/chains';
+
+import { getConfig } from '../config';
+
+const { PROJECT_ID } = getConfig();
+
+const getWallets = () => {
+  if (PROJECT_ID) {
+    return [injectedWallet, rainbowWallet, walletConnectWallet];
+  } else {
+    return [injectedWallet];
+  }
+};
+
+const connectors = connectorsForWallets(
+  [
+    {
+      groupName: 'Recommended',
+      wallets: getWallets(),
+    },
+  ],
+  {
+    appName: 'Web3 React boilerplate',
+    projectId: PROJECT_ID,
+  },
+);
+
+const config = createConfig({
+  chains: [localhost],
```

Comment:
would be nice to have a testnet chain too, like Sepolia

Reply:
1862fff